storage/fanout.go

```diff
@@ -429,7 +433,12 @@ type genericMergeSeriesSet struct {
 	sets []genericSeriesSet
 
 	currentSets []genericSeriesSet
-	querier     *mergeGenericQuerier
+
+	// 'querier' is optional and can be nil. Pass Querier if you want to retry query in case of failing series set.
```

Comment:
We never retry, why is this here?

Reply:
This is a moved comment from a factor function to the struct. I believe it's an outdated one, I'll remove.